### PR TITLE
azure-core, changelog for Apr2022 release

### PR DESCRIPTION
**Pull request by vhvb1989**

### CODE REVIEW

File: sdk/core/azure-core/CHANGELOG.md [hunk 1]
```diff
@@ -1,428 +1,428 @@
 # Release History
 
-## 1.5.0-beta.1 (Unreleased)
+## 1.5.0 (2022-03-31)
 
 ### Features Added
 
-- When a `RequestFailedException` exception is thrown, the `what()` method now includes information about the HTTP request which failed.
```
Comment (RickWinter) [inline sdk/core/azure-core/CHANGELOG.md:7]: Why the change from '-' to '*'

**Review by RickWinter — APPROVED** [review]

File: sdk/core/azure-core/CHANGELOG.md (hunk 1 above)
Comment (vhvb1989) [reply sdk/core/azure-core/CHANGELOG.md:7]: :(. Should've came from some format extension from VSCode..  I need to revert it
